_data/developer.yml

```diff
@@ -373,12 +373,10 @@ websites:
     url: https://cloud.mongodb.com
     img: mongodbmms.png
     tfa:
-      - sms
-      - phone
       - totp
-      - hardware
-    doc: https://docs.cloudmanager.mongodb.com/core/two-factor-authentication/
-    exception: "Phone 2FA for US / Canada only."
+      - email
+      - proprietary
```

Comment:
Are you sure they offer any proprietary software 2FA? The Okta thing sounds like a fork of Google Authenticatior.

Reply:
I had a look at the okta app on the [play store](https://play.google.com/store/apps/details?id=com.okta.android.auth&hl=en_GB&gl=US) and it appears to offer push notifications (see the third image), so assumed was proprietary